buildSrc/build.gradle

```diff
@@ -78,6 +78,9 @@ dependencies {
 
     if (localRepo) {
         compile name: "build-tools-${buildToolsVersion}"
+        // Required for dependency licenses task (explicitly added in case of localRepo missing transitive dependencies)
```

Comment:
I'm confused why this is necessary. Is the POM we publish to the local repo not including these dependencies?

Reply:
The POM isn't copied over into the local repo directory at the moment in any automation. If it were, would Gradle be able to pick these dependencies up?

Reply:
Perhaps, depends on how we are registering the repo. If we are using a flat dir repo then, no, as it doesn't handle metadata.